.env.devcontainer

```diff
@@ -1,21 +1,21 @@
 # set the port to listen on
-PORT=9000
+PORT=8000
 
 PYTHONDONTWRITEBYTECODE=1
 
 # drop into ipdb when breakpoint() is called
 PYTHONBREAKPOINT="ipdb.set_trace"
 
 SECRET_KEY='some-key'
-DATABASE_URL=mysql://root@127.0.0.1:3306/greencheck
-DATABASE_URL_READ_ONLY=mysql://root@127.0.0.1:3306/greencheck
+DATABASE_URL=mysql://root:deploy@db:3306/greencheck
```

Comment:
for the devcontainer, can either:

1. have all the containers on the same network where they can access each other at 127.0.0.1:<PORT_NUMBER>, but not access them from the host machine, which means we can't use any GUI tools installed on the host, **_or..._**
2. use the default compose networking, where we do not connect to localhost from _inside_ the containers, but instead use the service names instead. This _does_ allow us to connect from 'the outside' so using a graphical tool like way... [Sequel Ace](https://sequel-ace.com/) for exploring databases, and other tools to speak exposed services works.

I've chosen the latter here.